python/ray/tune/hyperband.py

```diff
@@ -30,33 +30,53 @@ class HyperBandScheduler(FIFOScheduler):
 
     Trials added will be inserted into the most recent bracket
     and band and will spill over to new brackets/bands accordingly.
+
+    See https://people.eecs.berkeley.edu/~kjamieson/hyperband.html
+
+    Args:
+        time_attr (str): The TrainingResult attr to use for comparing time.
+            Note that you can pass in something non-temporal such as
+            `training_iteration` as a measure of progress, the only requirement
+            is that the attribute should increase monotonically.
+        reward_attr (str): The TrainingResult objective value attribute. As
+            with `time_attr`, this may refer to any objective value. Stopping
+            procedures will use this attribute.
+        max_t_attr (int): maximum iterations per configuration. Altering
```

Comment:
max_t (int): max time units per trial. Trials will be stopped after this many time units (determined by time_attr) have passed. The HyperBand scheduler automatically tries to determine a reasonable number of brackets based on this and eta.

Reply:
Yeah it does - we can just keep the last one running then.